### DISCUSSION TITLE
New CL Argument

### DISCUSSION THREAD

**Discussion by zanderlewis** [elide-dev/elide#923] [Ideas]
What if we add a `elide previous` or something similar to revert back to a previous version of Elide? Elide is unstable and still in alpha and this feature could be useful when an update has a breaking change.

**Comment by sgammon** [comment]
There is a `selfupdate` command, but I don't know if it accepts an explicit version yet. That's a good idea. It should be easy to change versions of Elide as needed at least until it is stable.
